spec.html

```diff
@@ -137,6 +137,7 @@ <h1>Conformance</h1>
   <p>A conforming implementation of ECMAScript may provide additional types, values, objects, properties, and functions beyond those described in this specification. In particular, a conforming implementation of ECMAScript may provide properties not described in this specification, and values for those properties, for objects that are described in this specification.</p>
   <p>A conforming implementation of ECMAScript may support program and regular expression syntax not described in this specification. In particular, a conforming implementation of ECMAScript may support program syntax that makes use of any &ldquo;future reserved words&rdquo; noted in subclause <emu-xref href="#sec-keywords-and-reserved-words"></emu-xref> of this specification.</p>
   <p>A conforming implementation of ECMAScript must not implement any extension that is listed as a Forbidden Extension in subclause <emu-xref href="#sec-forbidden-extensions"></emu-xref>.</p>
+  <p>A conforming implementation of ECMAScript must not further define or redefine any facilities that are not implementation-defined, implementation-approximated, or host-defined.</p>
```

Comment:
This is a substantial new requirement - for example, it forbids adding new built-in functions anywhere except on the global object (so e.g. it disallows adding a new method to `JSON`). I doubt that's intentional; what were you trying to say? Maybe just the "redefine" part?

Reply:
Yes, just the redefine part. Will leave out "further define".